spec/bson/byte_buffer_read_spec.rb

```diff
@@ -101,6 +101,25 @@
     end
   end
 
+  describe '#get_uint32' do
+
+    let(:buffer) do
+      described_class.new("#{4294967295.to_bson.to_s}")
```

Comment:
I believe this can simply be `described_class.new(4294967295.to_bson)`.

Reply:
This sort of thing pops up in a bunch of other places, should I replace it all?

Reply:
I replaced what I could be replaced... Let me know what you think.

Reply:
What happens to this test if you use 4294967294?

Reply:
```1) BSON::ByteBuffer#get_uint32 gets the uint32 from the buffer
     Failure/Error: expect(int32).to eq(4294967295)
     
       expected: 4294967295
            got: 4294967294
     
       (compared using ==)
```

Reply:
i.e. it works if I change the expected output of the test

Reply:
If you change input to 4294967294 do you get 4294967294 back?

Reply:
Yes, I added a test for that...